src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c

```diff
@@ -68,10 +68,9 @@ typedef struct {
 ObjTsuboData sPotTypeData[4] = {
     { GAMEPLAY_DANGEON_KEEP, 0.197f, gameplay_dangeon_keep_DL_017EA0, gameplay_dangeon_keep_DL_018090, 12, 32,
       ObjTsubo_PotBreak1, ObjTsubo_PotBreak2, ObjTsubo_PotBreak3 },
-    { OBJECT_RACETSUBO, 0.29549998f, object_racetsubo_DL_000278, object_racetsubo_DL_001610, 18, 45,
-      ObjTsubo_RacePotBreak1, ObjTsubo_RacePotBreak2, ObjTsubo_RacePotBreak3 },
-    { OBJECT_TSUBO, 0.197f, object_tsubo_DL_0017C0, object_tsubo_DL_001960, 12, 32, ObjTsubo_PotBreak1,
-      ObjTsubo_PotBreak2, ObjTsubo_PotBreak3 },
+    { OBJECT_RACETSUBO, 0.29549998f, gGreenPotDL, gGreenPotShardDL, 18, 45, ObjTsubo_RacePotBreak1,
+      ObjTsubo_RacePotBreak2, ObjTsubo_RacePotBreak3 },
```

Comment:
It seems `GreenPot` has already been called `RacePot` in this file. Perhaps it should be called `RacePot` to maintain consistency?

Reply:
They show up on the top of clock tower when you contront skull kid tho, I think Greenpot is probably better.

Reply:
aughhh i just changed it lol wont that cause some confusion since theres gonna both green pot and race pot or idk maybe all of them should be green pot instead?

Reply:
Yeah they should probably all be green pot, IMO

Reply:
alright then ima wait for u guys to hash it out a bit lol then ill do what yall decide might be best

Reply:
Ah, I see. If it's on top of the clock town then yeah racePot isn't a good name. `greenPot` makes sense but I'm thinking it might be worth calling it `magicPot` instead, to align with `magicJar`, and `RacePot` -> `MagicPot` in obj_tsubo instead